docs/learn/concepts.md

```diff
@@ -1,52 +1,31 @@
+| [LYXt](../../docs/faq/lukso/general-information.md#whats-the-difference-between-lyxt-lyxe-and-lyx) | LYXt is the token for developers to use on our testnet. It can always be acquired by anyone for free.                                                                                                                                                                                                                                                                                                                             |
+| [LYXe](../../docs/faq/lukso/general-information.md#whats-the-difference-between-lyxt-lyxe-and-lyx) | LYXe is the token that was deployed on Ethereum before LUKSO Mainnet was launched. It allowed people to invest in LUKSO pre-launch.                                                                                                                                                                                                                                                                                               |
+| [Key Manager](../standards/universal-profile/lsp6-key-manager.md)                                  | The Key Manager is essentially a place where you can manage permissions that other _things_ have on your Universal Profile. _Things_ could be other people's profiles, dapps, extensions....anything.                                                                                                                                                                                                                             |
+| [Metadata](../standards/universal-profile/lsp3-profile-metadata.md)                                | This typically refers to the additional data that is stored against a token or NFT. The difference on LUKSO is it can be extended. <br></br> For example, imagine you've bought a house and have the deed as an NFT on LUKSO. Well now, as rennovations are carried out, they can be continually added to the NFT for history.                                                                                                    |
+| Account Abstraction                                                                                | Account abstraction is abstracting away smart contract execution from storing funds. With a smart contract account, you can program it to do all sorts of things. You can set up automatic payments, create multi-signature wallets that require multiple people to approve transactions, or even add security features like daily spending limits. It's like having a customizable, programmable bank account on the blockchain. |
+
+                                                                                                                                                                                                                                                                                                                                |
```

Comment:
this one seems unnecesary
<img width="894" alt="image" src="https://github.com/user-attachments/assets/47b7adf2-4ab8-4070-aa39-625fba83bb24">